### PR TITLE
Story 5

### PR DESCRIPTION
**Pull request by tylertomlinson**
User story 5 complete with tests passing 
coverage 99.8% (need to test index on `reviews controller`)

`ReviewsController`:
added `edit` and `update` methods

`Views`:
added `edit` view 
updated all files to have universal formatting 

added route for editing reviews 
testing for reviews edit functionally with edit page, required fields, filling in form
added flash message to be visible within all pages 
updated seed file to add review

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -13,9 +13,24 @@ def create
...
+
+  def update
+    review = Review.find(params[:id])
+    review.update(review_params)
+    if review.save
+      redirect_to "/shelters/#{review.shelter.id}"
+    else
+      flash[:notice] = "All fields are required"
+      redirect_to "/reviews/#{review.id}/edit"
+    end
```
Comment (StephanieFriend) [inline app/controllers/reviews_controller.rb:33]: Love this! Excited to see this implimented. :)

File: app/views/pets/edit.html.erb [hunk 2]
```diff
@@ -1,27 +1,20 @@
...
+	  <%= text_field_tag :name, @pet.name %><br/>
+	  <%= label_tag "Description:" %>
+	  <%= text_field_tag :description, @pet.description %><br/>
+	  <%= label_tag "Age (approx):" %>
+	  <%= text_field_tag :age, @pet.age%><br/>
+	  <%= label_tag "Sex:" %>
+	  <%= text_field_tag :sex, @pet.sex%><br/>
<br/>
+	<button type="submit">Submit</button>
+	<% end %>
```
Comment (StephanieFriend) [inline app/views/pets/edit.html.erb:18]: Really like the uniformity of the views

File: app/views/shelters/show.html.erb [hunk 3]
```diff
@@ -19,6 +24,6 @@
   <% if review.image != nil %>
     <%= image_tag review.image %>
   <% end %>
```
Comment (StephanieFriend) [inline app/views/shelters/show.html.erb:26]: Will need to add image size max

**Review by StephanieFriend — APPROVED** [review]
Really like these changes. The only thing is for us to implement the image max size.

File: app/views/shelters/show.html.erb (hunk 3 above)
Comment (tylertomlinson) [reply app/views/shelters/show.html.erb:26]: Yes. The seed file has a review image which some would say is too big

**Pull request merged**